acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/SecpSigningAcceptanceTest.java

```diff
@@ -17,13 +17,15 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.web3j.crypto.Sign.publicKeyFromPrivate;
 import static org.web3j.crypto.Sign.signedMessageToKey;
+import static tech.pegasys.web3signer.dsl.utils.MetadataFileHelpers.copyYubiHsmSimulator;
```

Comment:
do we need to add tests for Filecoin Signing with YubiHsm?

Reply:
I think we should.